src/routes/game/components/GameOverDialog.vue

```diff
@@ -24,11 +24,12 @@
       </template>
       <template v-if="currentMatch">
         <p v-if="currentMatch" class="dialog-text" data-cy="match-result-section">
-          Match against {{ gameStore.opponent.username }}
-          <span>: {{ matchIsOver ? 'Finished' : 'In Progress' }}</span>
+          <!-- Match against opponent: finished / in progress -->
+          {{ t('game.dialogs.gameOverDialog.matchAgainst') }} {{ gameStore.opponent.username }}
+          <span>: {{ t(matchIsOver ? 'game.dialogs.gameOverDialog.finished' : 'game.dialogs.gameOverDialog.inProgress') }}</span>
```

Comment:
I'm jiggering this to reduce the line length to fix that warning
```suggestion
          {{ t('game.dialogs.gameOverDialog.matchAgainst') }} {{ gameStore.opponent.username }}:
          <span>
            {{ t(matchIsOver ? 'game.dialogs.gameOverDialog.finished' : 'game.dialogs.gameOverDialog.inProgress') }}
            </span>
```